src/support/azext_support/aaz/latest/support/no_subscription/file/_list.py

```diff
@@ -170,17 +170,17 @@ def _build_schema_on_200(cls):
             )
 
             properties = cls._schema_on_200.value.Element.properties
-            properties.chunk_size = AAZFloatType(
+            properties.chunk_size = AAZIntType(
                 serialized_name="chunkSize",
             )
             properties.created_on = AAZStrType(
                 serialized_name="createdOn",
                 flags={"read_only": True},
             )
-            properties.file_size = AAZFloatType(
+            properties.file_size = AAZIntType(
```

Comment:
Check why type changed

Reply:
Because file size can only be int, this is correct